packages/analytics-browser/src/plugins/file-download-tracking.ts

```diff
@@ -0,0 +1,50 @@
+import { BrowserClient, PluginType, Event, EnrichmentPlugin } from '@amplitude/analytics-types';
+import { BrowserConfig } from '../config';
+
+const FILE_DOWNLOAD_EVENT = 'file_download';
```

Comment:
Probably discussed somewhere else too. Do we want to have a special prefix/name for this event, avoiding naming conflicts with customer-defined event?

Reply:
The event names are still TBD, and name will likely change! The placeholder name is inherited from a competitor. The plugin isn't auto added to the client yet so shouldn't be a concern yet.